### PR TITLE
bump rsvp

### PR DESCRIPTION
**Pull request by bekzod**

### CODE REVIEW

**Comment by rwjblue** [comment]
can you also update the package.lock?

**Comment by bekzod** [comment]
What is proper way of generating yarn.lock ? I do just yarn install thats all

**Comment by bekzod** [comment]
done

**Comment by bekzod** [comment]
just needs restart I think

**Comment by bekzod** [comment]
needs restart again

File: package.json [hunk 1]
```diff
@@ -56,7 +56,7 @@
     "handlebars": "^4.0.6",
     "jquery": "^3.2.1",
     "resolve": "^1.3.3",
-    "rsvp": "^3.5.0",
```
Comment (stefanpenner) [inline package.json]: 3.6.1, my bad

**Review by stefanpenner — CHANGES_REQUESTED** [review]

**Comment by bekzod** [comment]
green

**Pull request merged**